### PR TITLE
Add test for Ray outputs

### PR DESCRIPTION
**Pull request by suquark**
## Why are these changes needed?

This test checks that the output of ray is not polluted by other PRs.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: python/ray/tests/test_output.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        r"GiB memory available for workers and up to [0-9\.]+ GiB "
+        r"for objects. You can adjust these settings with .*?.", lines[0])
+    assert re.match(
+        logging_header +
+        r"services.py:\d+ -- View the Ray dashboard at .*?localhost:\d+?.*",
+        lines[1])
+
+
+if __name__ == "__main__":
+    import sys
```
Comment (ericl) [inline python/ray/tests/test_output.py:24]: I think you need to add a bazel rule that will run the test

File: python/ray/tests/test_output.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import re
+import pytest
+import ray
+
+
+def test_output(capsys, shutdown_only):
+    ray.init()
```
Comment (ericl) [inline python/ray/tests/test_output.py:7]: Should we sleep a bit to avoid race conditions?

Comment (edoakes) [reply python/ray/tests/test_output.py:7]: Alternatively could call it in a subprocess

Comment (suquark) [reply python/ray/tests/test_output.py:7]: subprocess could be better

**Comment by suquark** [comment]
I think our test is passing. All CI errors come from upstream.

**Review by ericl — APPROVED** [review]

**Pull request merged**